group/group_test.go

```diff
@@ -17,20 +17,34 @@ var allGroups = []group.Group{
 	group.Ristretto255,
 }
 
+const (
```

Comment:
I think these changes have made the code worse.

Reply:
My first thought as well, it looks worse now. The point of the const checker is to avoid logic errors due to typos, or prevent some form of repetition. That is not the case here, the labels are just used in tests and can be anything.

Reply:
I'm not sure the rule `goconst` is trying to enforce is actually a good idea in all cases.

Perhaps if you configure it with a larger `min-len` and `min-occurrences` then it's more reasonable?

https://golangci-lint.run/usage/linters/#goconst

Reply:
@mmcloughlin that is a good idea, adopting it.